content/blog/pointers-and-references-cpp/index.md

```diff
@@ -0,0 +1,153 @@
+﻿---
+{
+    title: 'Pointers and References in C/C++',
+    description: 'An overview of how pointers and references function in C/C++',
+    published: '2020-05-20T09:40:00.000Z',
```

Comment:
should these start to be release dated with the date it will be published so it doesn't have mismatching dates or is that handled somewhere in code based on merge date in Git logs?

Reply:
Honestly not sure about this one. A little unclear.

Reply:
We fix that by updating the date post-merge for when we plan to release